spec/support/helpers.rb

```diff
@@ -150,8 +152,8 @@ def gembin(cmd)
       ENV["RUBYOPT"] = old
     end
 
-    def sys_exec(cmd, expect_err = false)
-      Open3.popen3(cmd.to_s) do |stdin, stdout, stderr, wait_thr|
+    def sys_exec(cmd, expect_err = false, popen_opts = {})
+      Open3.popen3(cmd.to_s, popen_opts) do |stdin, stdout, stderr, wait_thr|
```

Comment:
:crying_cat_face:

I started using that pattern because I was getting warnings about nested chdir blocks. I can probably just avoid using the blocks.  I might not get back to this till next week though.